client/src/get-compiled-template.js

```diff
@@ -32,11 +23,20 @@ module.exports = function(cache){
             }
           });
         }
-        var type = template.type;
-        if (type === 'jade') { type = 'oc-template-jade'; }
-        if (type === 'handlebars') { type = 'oc-template-handlebars'; }
 
-        cb(null, templateEngines[type].getCompiledTemplate(templateText, template.key));
+        var type = template.type;
+        var ocTemplate;
+        try {
+          if (type === 'jade') { type = 'oc-template-jade'; }
+          if (type === 'handlebars') { type = 'oc-template-handlebars'; }
```

Comment:
can we move this ifs out of the try so it's clear the fail can happen only on the require line